packages/liferay-js-insights/index.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+const globby = require('globby');
+const findUp = require('find-up');
+const fs = require('fs');
+const util = require('util');
+const argv = require('minimist')(process.argv.slice(2));
+const parser = require('@babel/parser');
+const path = require('path');
+
+const readFile = util.promisify(fs.readFile);
+
+const INSIGHTS = [require('./insights/dependencies')];
+
+let report;
+
+if (argv.json) {
+	report = require('./reporters/json');
+} else if (argv.airtable) {
+	report = require('./reporters/airtable');
+} else {
+	report = require('./reporters/table');
+}
+
+/**
+ * Parses the contents of a file and returns the corresponding AST
+ */
+async function parse(path) {
+	const content = await readFile(path, 'utf8');
+
+	return parser.parse(content, {
+		plugins: ['jsx', 'classProperties'],
```

Comment:
Can these be alphabetized or do they need to apply in this order?

Reply:
Probably :)

Actually, I had to add `classProperties` because it's still present in some code in `segments-web`. Didn't we forbid that? Is it enabled by default in our current version in `npm-scritps`?

Reply:
It is enabled for everybody: https://github.com/liferay/liferay-npm-tools/blob/2d2f90a660b94a4b855eb7cb4912930be675146a/packages/liferay-npm-scripts/src/config/babel.json#L4